test/jit/test_pdt.py

```diff
@@ -224,3 +224,63 @@ def test_model_with_args(a, m):
         user_class = class_with_args(False)
         scripted_fn = torch.jit._script_pdt(test_model_with_args, example_inputs=[(10, user_class, ), (10.9, user_class, ), ])
         self.assertEqual(scripted_fn(100, class_with_args(True), ), test_model_with_args(100, class_with_args(True)))
+
```

Comment:
I think there is two missing test cases here:
1) methods on `nn.Module` that are exported by `torch.jit.export`
2) methods that are recursively invoked by `forward`

Reply:
For 1.) This blocked on Inferring types of those functions which are not called under the hood of MonkeyType. 

2.) This one is addressed.